openfisca_us/parameters/tax/ALD/misc/employer_share.yaml

```diff
@@ -0,0 +1,5 @@
+description: Employer share of self-employment tax
+values:
+  0000-01-01: 0.5
+metadata:
+  unit: USD
```

Comment:
not USD (several other parameters are also proportions)

Reply:
Done